src/server/graphql/models/CachedUser/cachedUserMutation.js

```diff
@@ -1,11 +1,11 @@
-import r from '../../../database/rethinkdriver';
+import r from '../../../database/rethinkdriver'; // eslint-disable-line id-length
 import {GraphQLString} from 'graphql';
 import {CachedUser} from './cachedUserSchema';
-import {getUserByUserId} from './helpers';
+// import {getUserByUserId} from './helpers';
```

Comment:
I favored commenting out unused imports because they were generally auth related (not just in this case), and I figured they could have been added for future use - also related to the TODO below.